rmgpy/data/thermo.py

```diff
@@ -1202,26 +1208,32 @@ def get_thermo_data(self, species, training_set=None):
         The method corrects for symmetry when the molecule uses machine
         learning or group additivity. Libraries and direct QM calculations
         are already corrected.
+
+        If either metal to scale to or from is not specified, assume the binding energies given in the input file
         
         Returns: ThermoData
         """
         from rmgpy.rmg.input import get_input
 
         thermo0 = self.get_thermo_data_from_libraries(species)
 
-        if thermo0 is not None:
+        if thermo0 is not None:  # was able to find thermodata in the loaded libraries
             if len(thermo0) != 3:
                 raise RuntimeError("thermo0 should be a tuple (thermo_data, library, entry), not {0}".format(thermo0))
+            entry = thermo0[2]
             thermo0 = thermo0[0]
 
             if species.contains_surface_site():
-                thermo0 = self.correct_binding_energy(thermo0, species)
+                if entry.metal is not None:
+                    thermo0 = self.correct_binding_energy(thermo0, species, metal_to_scale_from=entry.metal, metal_to_scale_to=metal_to_scale_to)
```

Comment:
I think we are missing the facet here since the facet is stored as entry.facet.  So for a Pt111 entry, entry.metal = Pt, so we scale from Pt not Pt111

Reply:
fixed!